.gitignore

```diff
@@ -57,7 +57,7 @@ playground.xcworkspace
 # Pods/
```

Comment:
I would still add this like as well! As mention in the .gitignore, it's up to us whether we want to keep it or remove it. IMO it would be worth while to put pods in gitignore because this is an opensource project and one of the pros of gitigbotring is

> There won't be any conflicts to deal with when performing source control operations, such as merging branches with different Pod versions.

And that might be a problem when dealing with multiple branches and more people working on the repo

Reply:
Okay, I will add it!